### PR TITLE
Fix carla speed calculation bug and add success info

### PR DESCRIPTION
**Pull request by emailweixu**
Also resize image for rendering.

### CODE REVIEW

File: alf/environments/suite_carla.py [hunk 1]
```diff
@@ -75,6 +75,10 @@ def is_available():
     return carla is not None
 
 
+MINIMUM_RENDER_WIDTH = 640
+MINIMUM_RENDER_HEIGHT = 240
```
Comment (hnyu) [inline alf/environments/suite_carla.py:79]: why changing from 480 to 240?

File: alf/environments/suite_carla.py [hunk 2]
```diff
@@ -1023,10 +1035,13 @@ def get_current_time_step(self, current_frame):
                                                 self._actor.get_location())
         v = self._actor.get_velocity()
         obs['speed'] = np.float32(
-            np.float32(v.x * v.x + v.y * v.y + v.z * v.z))
+            np.sqrt(np.float32(v.x * v.x + v.y * v.y + v.z * v.z)))
```
Comment (hnyu) [inline alf/environments/suite_carla.py:1038]: Is this a serious issue? It seems that in principle before and after are equivalent.

File: alf/environments/suite_carla.py (hunk 1 above)
Comment (emailweixu) [reply alf/environments/suite_carla.py:79]: The height of text panel is 240. There is no need beyond that.

File: alf/environments/suite_carla.py (hunk 2 above)
Comment (emailweixu) [reply alf/environments/suite_carla.py:1038]: The range would huge without the sqrt. I thought it shouldn't affect the training much. But somehow the training is better after fixing this. Although one training is not enough to draw the conclusion.

**Review by hnyu — APPROVED** [review]

**Pull request merged**